my_account_helper/model/xmlrpc_helper.py

```diff
@@ -0,0 +1,49 @@
+﻿# -*- encoding: utf-8 -*-
+#
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emmanuel Mathier
+#
+#    The licence is in the file __openerp__.py
+#
+
+from openerp.osv import orm
+
+
+class xmlrpc_helper(orm.Model):
+
+    _inherit = "res.partner"
+
+    # return all child information for all childs from a partner_id specified

```

Comment:
Comments for methods are done with docstrings (3 double quotes) placed after the def statement.